docs/api/features/get.asciidoc

```diff
@@ -1,29 +1,28 @@
 [[features-api-get]]
-=== Get Features
-
-experimental[This API is *experimental* and may be changed or removed completely in a future release.]
+=== Get features API
+++++
+<titleabbrev>Get features</titleabbrev>
+++++
 
 Retrieves all {kib} features.
 
-Note: You cannot access this endpoint via the Console in {kib}.
+experimental[This API is *experimental* and may be changed or removed completely in a future release.]
 
-==== Get all {kib} features
+[[features-api-get-request]]
+==== Request
 
-===== Request
+`GET /api/features/v1`
```

Comment:
__issue:__ @KOTungseth  it should be `/api/features`

Reply:
@azasypkin just to clarify, does this change apply only to 7.x, or does it apply to other versions too?

Reply:
@KOTungseth only 7.x/7.5+ (that is, 7.x and master branches)